manifests/0000_60_cluster-platform-operator-manager_01-rukpak-webhook.certificate.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: platform-operators-rukpak-webhook-certificate
+  namespace: openshift-platform-operators-system
```

Comment:
nit: if this were the namespace, I think we would drop the `system` suffix to be idiomatic with other openshift namespaces.

Reply:
This is the right namespace I believe as it is just using the namespace that other platform operators manifests use. That being said, we could look at dropping the `system` suffix to be more idiomatic.

Reply:
I'm not sure if `system` suffix's are less idiomatic now that new components are being bootstrapped with this in mind. I'd be surprised if you fired up a newer 4.x version, and there's no namespaces with this system suffix tbh.

Reply:
Yeah, fair enough. I'll boot up a 4.x cluster just to make sure we're in the green here wrt to openshift idioms.

Reply:
So looking at a 4.11 cluster I found no default OpenShift namespaces that had a `-system` suffix.

Reply:
Yes, I would recommend dropping `system` and having the namespace be `openshift-platform-operators` to be more idiomatic with other openshift namespaces

Reply:
Let me know how we feel about this. I have some changes standing-by that remove the `-system` suffix. If we want to go down that path I can go ahead and push it up. Additionally, I couldn't find anything that has a restriction on how namespaces are named other than [this section](https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#api:~:text=It%20is%20important%20to%20note,be%20annotated%20with%20the%20label%3A) that talks about namespace run levels.

Reply:
Yup, works for me.

Reply:
Alrighty, those changes are up.

Reply:
Sounds good :+1: